user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+statistics to analyze the efficiency of the campaign. 
+
+All of these operations will only be available within OroCRM if the capability has been set to *"System"*.
+
+The MailChimp intergation opportunities are usually  used by marketing associates and other managers responsible for 
+email campaigns.
+
+
+.. _admin-capabilities-config-entities:  
+
+Manage Configurable Entities
+----------------------------
+
+Many entities in OroCRM can be configured from the UI, as described in the
+:ref:`Entities guide <user-guide-entity-management-from-UI>`. The user can change the attachments settings, 
+define whether the entity should be displayed on a grid and/or a View page, whether it will be 
+exported to a .CSV file, and define other settings. For some of them, it is also possible to add new fields, as 
+described in the :ref:`Entity Fields guide <user-guide-field-management>`. 
+
+These actions are only available if the *"Manage Configurable Entities"* capability is set to *"System"*. 
+
+They are usually performed by the system administrators.
+
+
+.. _admin-capabilities-org-calendar-events: 
+
+Manage Organization Calendar Events
+-----------------------------------
+
+If this capability has been set to *"System"*, the user create, edit, and delete events in organization-wide calendars, 
+which are described in more detail in the :ref:`corresponding section <user-guide-calendars-system>` of 
+the *Calendars Overview* guide.
+
+Organization calendar events are usually managed by organization-level managers and HRs.
+
+.. hint::
+
+     Even if this capability is set to *"None"*, the users can still view organization-wide calendars, add 
+     them to their own calendar views, and copy related events to their own calendars.
+
+
+.. _admin-capabilities-sys-calendar-events: 
+
+Manage System Calendar Events
+-----------------------------
+
+If this capability has been set to *"System"*, the user create, edit, and delete events in system-wide calendars, which 
+are described in more detail in the <user-guide-calendars-system>` of the *Calendars Overview* guide.
+
+System calendar events are usually managed by the company managers and HRs.
+
+.. hint::
+
+     Even if this capability is set to *"None"*, the users can still view organization-wide calendars, add them to their 
+     own calendar views, and copy related events to their own calendars.
+
+
+.. _admin-capabilities-sys-calendars: 
+
+Manage System Calendars
+-----------------------
+
+
+If this capability has been set to *"System"*, the users can 
+:ref:`create <user-guide-calendars-system>` and :ref:`manage <user-guide-calendars-manage>` system-wide calendars.
+
+System-wide calendars are usually created and managed by system administrators and top managers.
+
+
+.. _admin-capabilities-passwords:
+ 
+Manage Users' Passwords
+-----------------------
+
+If the capability is set to *"System"*, the user can change passwords of other users. Usually, this is only done
```

Comment:
Edit: If the capability is set to *"System,"* the user can change passwords of other users. Usually, this is only done